utilities/backupable/backupable_db.cc

```diff
@@ -386,9 +387,9 @@ class BackupEngineImpl {
       app_metadata_ = app_metadata;
     }
 
-    Status AddFile(std::shared_ptr<FileInfo> file_info);
+    IOStatus AddFile(std::shared_ptr<FileInfo> file_info);
```

Comment:
I agree with your original assessment that AddFile and Delete here are not IO functions, only LoadFromFile and StoreToFile.

Reply:
> I agree with your original assessment that AddFile and Delete here are not IO functions, only LoadFromFile and StoreToFile.

Should I change back to "Status"?

Reply:
If you want to. Not a big deal